From 790dc229a9ce193edbcc93aae44f644015fce3b8 Mon Sep 17 00:00:00 2001
From: Russ Weight <russell.h.weight@intel.com>
Date: Wed, 9 Mar 2016 16:23:23 -0800
Subject: [PATCH 2/2] usb: cdc-acm: REVERTME: whitespace changes

This patch removes some minor white-space changes between the quilt and
the git source. The code is actually more correct without this patch,
so it may be best update the git source in the near future and then
remove this patch from the quilt.

Change-Id: I8f3486ae8cbd67989957a26bd7830481751b1e64
Signed-off-by: Russ Weight <russell.h.weight@intel.com>
---
 drivers/usb/class/cdc-acm.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/usb/class/cdc-acm.c b/drivers/usb/class/cdc-acm.c
index e37d4f1..c7d0d1c 100644
--- a/drivers/usb/class/cdc-acm.c
+++ b/drivers/usb/class/cdc-acm.c
@@ -885,7 +885,7 @@ static int wait_serial_change(struct acm *acm, unsigned long arg)
 		}
 	} while (!rv);
 
-
+	
 
 	return rv;
 }
@@ -1047,8 +1047,7 @@ static int acm_write_buffers_alloc(struct acm *acm)
 }
 
 /* Zero packet needs to be sent for some modems */
-static bool acm_needs_zerp_pkt(struct usb_device *usb_dev)
-{
+static bool acm_needs_zerp_pkt(struct usb_device *usb_dev) {
 	return usb_dev->descriptor.idVendor == INTEL_MODEM_VID &&
 		usb_dev->descriptor.idProduct == INTEL_MODEM_PID;
 }
-- 
1.9.1

